features-json/beforeafterprint.json

```
@@ -1,7 +1,7 @@
 {
   "title":"Printing Events",
   "description":"Window fires `beforeprint` and `afterprint` events so the printed document can be annotated.",
-  "spec":"https://www.w3.org/TR/html5/webappapis.html#printing",
+  "spec":"https://html.spec.whatwg.org/multipage/timers-and-user-prompts.html#printing",
   "status":"rec",
```

```suggestion
  "status":"ls",
```
This should fix your PR 🙂
More reading [here](https://github.com/Fyrd/caniuse/blob/master/CONTRIBUTING.md).

---

```diff
-  "status":"rec",
+  "status":"ls",
```
> This should fix your PR :)
> More reading [here](https://github.com/Fyrd/caniuse/blob/master/CONTRIBUTING.md).

Thanks — much appreciated